app/org/sagebionetworks/bridge/models/notifications/NotificationRegistration.java

```diff
@@ -0,0 +1,76 @@
+package org.sagebionetworks.bridge.models.notifications;
+
+import org.sagebionetworks.bridge.dynamodb.DynamoNotificationRegistration;
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(as=DynamoNotificationRegistration.class)
+public interface NotificationRegistration extends BridgeEntity {
+
+    public static NotificationRegistration create() {
+        return new DynamoNotificationRegistration();
+    }
+    
+    /**
+     * The healthCode of the user registering their device for notifications.
+     */
+    public String getHealthCode();
+    
+    /** @see #getHealthCode */
+    public void setHealthCode(String healthCode);
+
+    /**
+     * The GUID of this registration that is returned to the client to update the 
+     * device identifier in subsequent requests. Can also be used to unregister 
+     * for notifications. 
+     */
+    public String getGuid();
+    
+    /** @see getGuid */
+    public void setGuid(String guid);
+    
+    /**
+     * The platformEndpoint ARN generated when the user's device is registered 
```

Comment:
platformARN is an identifier representing the set-up of a notifications service like APMS for a specific application. What SNS calls either the platformEndpointARN or the endpointARN, and which I decided I would internally call the endpointARN, is a registration between a specific device, and the platformARN. That's the thing we need to send a push notification to an individual device. It's what gets registered with a topic as well. So I will change the above "platformEndpoint ARN" to match this terminology.